### ISSUE TITLE
Nightscout not updating properly

### ISSUE THREAD

**Issue by GBRacerX1** [LoopKit/Loop#1484]
**Describe the bug**
Latest 14.1 version of nightscout as well as version below not upating properly from loop.  The loop pill is not updating consistently.  Also the overrides are not showing in nightscout as well as the Terence arrow. 

**Attach an Issue Report**
Tap the Loop settings icon on the bottom of the screen, then tap Issue Report and attach it to this ticket.

**To Reproduce**
1. Log into nightscout instance
2. Loop pill will go stale and show loop is not looping even though it is
3. Create or select an override
4. Nightscout does not update with override
5. Nightscout does not show trend arrow 

**Expected behavior**
Nightscout loop pill is up to date every time loop loops
Nightscout will show loop overrides selected
Nightscout will show trend arrow for reading 

**Phone**
 - Hardware: iphone xr plus
 - OS Version: 14.4

**Loop Version**
 - Version Number: Dev latest
 - Repo: [LoopKit/Loop]

**CGM**
 - Device: [Dexcom G6]
 - Manager app: [Dexcom App]

**Pump**
 - Manufacturer: Omnipod 
 - Model: [Eros]

**Comment by loudestnoise** [comment]
I've also noticed for my Nightscout that is being fed by Dexcom Share (via bridge in Nightscout) and DIY Loop `dev` branch that my SGV lacks the arrow. 
![image](https://user-images.githubusercontent.com/194853/107815711-5dd90800-6d39-11eb-9241-59ac856af77f.png)

**Comment by GBRacerX1** [comment]
I forgot to add the arrow in the bug. Nightscout seems to broken with the new service approach. It’s inconsistent with the uploading.

**Comment by loudestnoise** [comment]
I'm wondering if despite not having toggling `Upload Readings` under `Remote Data Synchronization` if `dev` Loop is still doing so. I started to realize this as yesterday my Dexcom was reading higher than I felt, so I checked with my OneDrop meter, which syncs over Bluetooth to my iPhone and is written to Apple Health. I see that reading on my Nightscout, despite AFAIK only having the Nightscout bridge set up as my CGM source. Also apparently OneDrop is logging the same reading three times in my Apple Health, which is uh, weird.
![image](https://user-images.githubusercontent.com/194853/110040526-95b7e780-7d08-11eb-9f08-5fe449e56f07.png)
